doc/assets/xml_spec.md

```diff
@@ -192,7 +200,15 @@ always present
       <xs:element name="location" minOccurs="0"></xs:element>
       <xs:element name="type" type="xs:string" minOccurs="0"></xs:element>
       <xs:element name="full_lhs" type="xs:string"></xs:element>
-      <xs:element name="full_lhs_value" type="xs:int"></xs:element>
+      <xs:element name="full_lhs_value">
+        <xs:complexType>
+          <xs:simpleContent>
+            <xs:extension base="xs:string">
+              <xs:attribute name="binary" type="xs:string"/>
```

Comment:
:pick: this could be made more precise using a XSD type where we limit this to be only [01]+